src/pkg/packaging-tools/sharedFramework/sharedFramework.csproj

```diff
@@ -0,0 +1,153 @@
+<Project Sdk="Microsoft.NET.Sdk" InitialTargets="GetPackageReference" DefaultTargets="Restore;Publish">
+  <!--
+    This project lays out a shared framework using the SDK targets.
+  -->
+
+  <UsingTask TaskName="ProcessSharedFrameworkDeps" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
+    <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <PackageId>unused</PackageId>
+
+    <ProjectDepsFileName>$(SharedFrameworkName).deps.json</ProjectDepsFileName>
+
+    <MicrosoftNETPlatformLibrary>do-not-consider-existing-platform</MicrosoftNETPlatformLibrary>
```

Comment:
Can you add a comment?

Reply:
Will do. This was for producing Microsoft.NETCore.App on the new infra which I didn't end up finishing for this PR.